_posts/2019-04-07-sr2019-tlc-crowned-champion.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: news
+title: Team TLC Wins Student Robotics 2019
+---
+
+{% include figure.html
+           src="/images/content/news/sr2019/sr2019-photo.jpg"
```

Comment:
This image is _huge_ -- about an order of magnitude larger than needed. Please can we resize it to be no larger than 1500px in either dimension. I would suggest that even 1000px wide is plenty.

The current image weight in this article is about 8.4M, which is a lot given that this article will be on our homepage for a considerable time.

If we want to keep the higher resolution images around and link to them that's fine, but the embeds should ideally be at the size they're going to be rendered at (or maybe 2x that if we want to bias towards high-dpi clients).